source/Dialog.h

```diff
@@ -44,13 +44,12 @@ class Dialog : public Panel {
 	// Three different kinds of dialogs can be constructed: requesting numerical
 	// input, requesting text input, or not requesting any input at all. In any
 	// case, the callback is called only if the user selects "ok", not "cancel."
-template <class T>
-	Dialog(T *t, void (T::*fun)(int), const std::string &text);
```

Comment:
Keep this template. While it isn't used right now, template code that isn't used is not included in the binary.

(This method was originally used by BankPanel before [support for shorthand numbers like "800k" was added](https://github.com/endless-sky/endless-sky/commit/8fb7692023ac4b0ca6f55de7a3c5ddbbc8c8a134).)

Reply:
Done. Thanks for the reference.